### PR TITLE
MEPTS-1109 | TX PVLS changes December Release

### PR DESCRIPTION
**Pull request by iteliolucas**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly26.java [hunk 1]
```diff
@@ -121,22 +121,7 @@ public ReportDefinition constructReportDefinition() {
...
-    rd.addDataSetDefinition("R", Mapped.mapStraightThrough(txRttDataset.constructTxRttDataset()));
-    rd.addDataSetDefinition(
-        "T", Mapped.mapStraightThrough(transferredInDataset.constructTransferInDataset()));
-    rd.addDataSetDefinition(
-        "PL", Mapped.mapStraightThrough(txRTTPLHIVDateset.constructTxRTTPLHIVDateset()));
-    rd.addDataSetDefinition(
-        "PREP", Mapped.mapStraightThrough(prepNewDataset.constructPrepNewDataset()));
-    rd.addDataSetDefinition("DT", Mapped.mapStraightThrough(new DatimCodeDatasetDefinition()));
-    rd.addDataSetDefinition(
-        "PREPNUM", Mapped.mapStraightThrough(prepCtDataset.constructPrepCtDataset()));
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly26.java]: Retornar

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly26.java [hunk 2]
```diff
@@ -38,8 +37,9 @@
 import org.openmrs.module.reporting.report.ReportDesign;
 import org.openmrs.module.reporting.report.definition.ReportDefinition;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
 
-@Deprecated
+@Component
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly26.java]: Deprecated

**Review by psmatsinhe — COMMENTED** [review]
Have a look

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly26.java (hunk 2 above)
Comment (iteliolucas) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupMERQuarterly26.java]: Done

**Review by eliazmuchave — APPROVED** [review]
Looks good to me

**Review by eliazmuchave — APPROVED** [review]

**Pull request merged**